src/printer.js

```diff
@@ -65,12 +64,14 @@ Printer.prototype.print = function(callback) {
 	async.eachSeries(
 		_self.commandQueue,
 		function(command, callback) {
-			if (_self.commandDelay !== 0) {
-				sleep.usleep(_self.commandDelay);
+			function write() {
+				_self.serialPort.write(command, function() {
+					_self.serialPort.drain(callback);
+				});
 			}
-			_self.serialPort.write(command, function() {
-				_self.serialPort.drain(callback);
-			});
+
+			if (_self.commandDelay === 0) return write()
```

Comment:
`setTimeout(..., 0)` still produces a timeout of a few millisecond due to how scheduling works in javascript. I think that it would be best to just let it write as soon as possible.